--- groff-1.22.2/src/devices/grops/ps.cpp~	2013-02-07 12:06:07.000000000 +0000
+++ groff-1.22.2/src/devices/grops/ps.cpp	2017-01-14 23:37:57.641591670 +0000
@@ -1386,6 +1386,7 @@
      .comment_arg(Version_string)
      .end_comment();
   {
+    char *en;
     fputs("%%CreationDate: ", out.get_file());
 #ifdef LONG_FOR_TIME_T
     long
@@ -1393,6 +1394,9 @@
     time_t
 #endif
     t = time(0);
+    en = getenv("SOURCE_DATE_EPOCH");
+    if (en)
+	t = strtoull(en, NULL, 10);
     fputs(ctime(&t), out.get_file());
   }
   for (font_pointer_list *f = font_list; f; f = f->next) {
--- groff-1.22.2/src/devices/grohtml/post-html.cpp~	2013-02-07 12:06:07.000000000 +0000
+++ groff-1.22.2/src/devices/grohtml/post-html.cpp	2017-01-14 23:38:43.591921827 +0000
@@ -5014,11 +5014,6 @@
 	.put_string(Version_string)
 	.end_comment();
 
-      t = time(0);
-      html.begin_comment("CreationDate: ")
-	.put_string(ctime(&t), strlen(ctime(&t))-1)
-	.end_comment();
-
       if (dialect == html4)
 	writeHeadMetaStyle();
 
@@ -5127,10 +5122,6 @@
     .put_string(Version_string)
     .end_comment();
 
-  t = time(0);
-  html.begin_comment("CreationDate: ")
-    .put_string(ctime(&t), strlen(ctime(&t))-1)
-    .end_comment();
 
   if (dialect == html4)
     writeHeadMetaStyle();
